var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -142,6 +142,8 @@ def cmake_enable_option(spec_requiremement):
                                                              "+rx3d",
                                                              "+coreneuron",
                                                              "+tests"]]
+        if '+tests' in self.spec:
+            args.append('-DNRN_ENABLE_TESTS_BBP:BOOL=ON')
```

Comment:
Are we going to open-source that at some point?

Reply:
I believe so. Once it's open sourced then we can change things so it is enabled by `NRN_ENABLE_TESTS`. But I think we can anticipate that we will often have models that are "not open source yet"... 🤷